h/static/scripts/frame-rpc.js

```diff
@@ -34,81 +34,81 @@ var VERSION = '1.0.0';
 module.exports = RPC;
 
 function RPC (src, dst, origin, methods) {
```

Comment:
This was adapted from third-party code so might be worth excluding from this change.